README.md

```diff
@@ -211,6 +212,9 @@ Share your apps here! Submit a pull request!
 
 * **Appwrite + Angular SimpleGrocery** - A simple app to create fast shopping-list. [Source Code](https://github.com/ffex/simple-grocery-demo)
 
+### Cordova
+* **Appwrite + Cordova + React** - A simple instagram clone built with Appwrite, Cordova and React (with vite). [Source Code](https://github.com/arnu515/instagram-clone-with-cordova-appwrite)
```

Comment:
I tried running your project but got an error:

```
$ npx cordova run electron --nobuild
Current working directory is not a Cordova-based project.
```

Do you know what's wrong? Maybe I'm missing files like the files in the `www` folder?

Reply:
I believe you have to run `yarn build` first, so that the `www` folder gets created. I'll add it to the README